### ISSUE TITLE
Adding nodes after initialization?

### ISSUE THREAD

**Issue by dakipro** [500tech/angular-tree-component#224]
Hi,

On previous version I was able to define nodes like this
`categoriesTree: Category[];`

and then after getting data from server, to assign them to the tree
`this.categoriesTree = dataFromServer; //array of Categories`

This worked fine before I updated the library. Now only way I can get the component to show anything is only by assigning some value initially, 
```
categoriesTree = [{id: 1,
          name: "name"}]

```
and then later updating/changing assigned data. 
```

this.categoriesTree = [{id: 2,
          name: "name2"}]

```
Which expects me to always have the data before initializing the component (very difficult to handle).

Am I doing something basically wrong here? How can I initialize component and then give it the data it should display? I tried calling the update, but couldn't find clear instructions on how to achieve it.

Thanks

**Comment by dojchek** [comment]
@dakipro 
Did you try to do this with:
`TreeModel.setData({ nodes, options = null, events = null })`

I'm successfully setting the nodes up with this method.

**Comment by adamkleingit** [comment]
Yeah it's a bug and will be solved in the coming version

**Comment by adamkleingit** [comment]
Hi @dakipro ,
I just published 3.2.4 and it works for me there - can you verify?
Thanks
